src/reusedBrowser.ts

```diff
@@ -78,11 +78,11 @@ export class ReusedBrowser implements vscodeTypes.Disposable {
     this._disposables = [];
   }
 
-  private async _startBackendIfNeeded(config: TestConfig): Promise<{ errors?: string[] }> {
-    // Unconditionally close selector dialog, it might send inspect(enabled: false).
+  private async _startBackendIfNeeded(config: TestConfig) {
     if (this._backend) {
+      // cancel any ongoing recording and hide toolbar, so the caller of _startBackendIfNeeded gets a clean slate.
```

Comment:
IIRC, we discussed that this is here to stop recording/picking locator before we run/debug tests. I think we've also agreed that we should probably move this to `willRunTests` or similar.